### PR TITLE
docs/rkt: Document the different UX.

### PR DESCRIPTION
**Pull request by yifan-gu**
As for https://github.com/kubernetes/kubernetes/pull/20905#issuecomment-183419853

cc @kubernetes/sig-node @jonboulle @sjpotter

### CODE REVIEW

File: docs/getting-started-guides/rkt/notes.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+<!-- END STRIP_FOR_RELEASE -->
+
+<!-- END MUNGE: UNVERSIONED_WARNING -->
+
+# Notes on Different UX with rkt container runtime
+
+### Doesn't support ENTRYPOINT + CMD feature
+
+To run a Docker image, rkt will transfer it into App Container Image (ACI) format first.
```
Comment (yujuhong) [inline docs/getting-started-guides/rkt/notes.md]: s/transfer/convert ?

File: docs/getting-started-guides/rkt/notes.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+However, during the transformation, the `ENTRYPOINT` and `CMD` are concatentated to construct ACI's `Exec` field.
```
Comment (yujuhong) [inline docs/getting-started-guides/rkt/notes.md]: s/transformation/conversion

File: docs/getting-started-guides/rkt/notes.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+This means after the transformation, we are not able to replace only `ENTRYPOINT` or `CMD` without touching the other part.
+So for now, users should specify the **full executable path + arguments** in either the container spec's `Command` or `Args`.
```
Comment (yujuhong) [inline docs/getting-started-guides/rkt/notes.md]: Specifying full path + args in `Args` seems a big strange to me. Shouldn't we tell the users to set both `Command` and `Args`?

File: docs/getting-started-guides/rkt/notes.md (hunk 1 above)
Comment (yujuhong) [inline docs/getting-started-guides/rkt/notes.md]: Link to ACI format?

**Comment by yifan-gu** [comment]
@yujuhong Addressed comments, ptla, thanks :)

File: docs/getting-started-guides/rkt/notes.md [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+spec:
+  containers:
+  - name: busybox
+    image: busybox
+    args:
+    - /bin/sleep
+    - 1000
+```
+
+All the three examples above are valid as they contains the full executable path, plus the arguments.
```
Comment (yujuhong) [inline docs/getting-started-guides/rkt/notes.md]: s/contains/contain

File: docs/getting-started-guides/rkt/README.md [hunk 5]
```diff
@@ -206,6 +206,9 @@ See [a simple nginx example](../../../docs/user-guide/simple-nginx.md) to try ou
 
 For more complete applications, please look in the [examples directory](../../../examples/).
 
+### Different UX with rkt container runtime
+
+rkt and Docker has a very different design, as well as ACI and Docker image format. Users might experience some different experience when switching from one to the other. More information can be found [here](notes.md).
```
Comment (yujuhong) [inline docs/getting-started-guides/rkt/README.md]: s/has a very different design/have very different designs

**Comment by yujuhong** [comment]
lgtm with some nits.

/cc @dchen1107

**Comment by yifan-gu** [comment]
@yujuhong Comments fixed. Thank you :)
cc @jonboulle for another pair of eyes.

**Pull request merged**